### PR TITLE
KEP-2133: update doc references for KubeletCredentialProviders

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <andrewsy@google.com>

https://github.com/kubernetes/enhancements/issues/2133

### CODE REVIEW

File: content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: Configure the kubelet's image credential provider plugin
 content_type: task
 ---
 
-{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
 <!-- overview -->
```
Comment (sftim) [inline content/en/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md:12]: This is good to go. With caveats.

---

Ideally, open a follow up PR and in that PR:
- write the first occurrence of _exec plugins_ in italics
- consider rewording that italicized text to _credential helper plugins_
- tweak the text
  > a single subdomain segment, so `*.io` does NOT match `*.k8s.io`.

  and change it to use bold, not caps, for emphasis, like this
  ```markdown
  a single subdomain segment, so `*.io` does **not** match `*.k8s.io`.
  ```

---

Is this still the latest (alpha) version:
```
apiVersion: kubelet.config.k8s.io/v1alpha1
kind: CredentialProviderConfig
```
?

If not, we need a few updates.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -281,6 +279,9 @@ For a reference to old feature gates that are removed, please refer to
 | `IndexedJob` | `false` | Alpha | 1.21 | 1.21 |
 | `IndexedJob` | `true` | Beta | 1.22 | 1.23 |
 | `IndexedJob` | `true` | GA | 1.24 | - |
+| `KubeletCredentialProviders` | `false` | Alpha | 1.20 | 1.23 |
+| `KubeletCredentialProviders` | `true` | Beta | 1.24 | 1.26 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:283]: nit / needs fixing really
```suggestion
| `KubeletCredentialProviders` | `true` | Beta | 1.24 | 1.25 |
```

**Review by sftim — COMMENTED** [review]
Almost LGTM, but I strongly recommend that we also update https://kubernetes.io/docs/concepts/containers/images/#configuring-nodes-to-authenticate-to-a-private-registry to link to the kubelet credential provider setup task page.

**Comment by andrewsykim** [comment]
Closing in favor of https://github.com/kubernetes/website/pull/38006

/close

**Pull request closed without merging**